src/styles/blocks/main.scss

```
@@ -0,0 +1,17 @@
+.main {
+  overflow: hidden;
+  background-color: #fff;
+  @media (max-width: 1024px) {
+    // overflow: hidden;
+    padding: 0 50px;
+    // background-color: $hover;
```

```suggestion
    padding: 0 50px;
```
Too many commented lines left